base/rico-core/src/main/resources/build.properties

```diff
@@ -15,6 +15,6 @@
 # limitations under the License.
 #
 
-version=$version
-buildDate=$buildDate
-buildTime=$buildTime
```

Comment:
I'd suggest to leave the variables in here such that the repository does not contain concrete value, only during build are they set. And we include build.properties to .gitignore so that the update in it do not appear in our IDE changelist.

Reply:
I also thought about that but still committed with versions - basically we can  leave them empty.